shimmy/meltingpot_compatibility.py

```diff
@@ -42,41 +43,28 @@ class MeltingPotCompatibilityV0(ParallelEnv, EzPickle):
 
     def __init__(
         self,
-        substrate_name: str,
+        env: meltingpot.python.utils.substrates.substrate.Substrate,
```

Comment:
Shouldn't this type hint include None and default to None

Reply:
Oh good catch, missed that